WalletWasabi/Cache/IdempotencyRequestCache.cs

```diff
@@ -75,13 +75,13 @@ public IdempotencyRequestCache(IMemoryCache cache)
 					ResponseCache.Remove(request);
 				}
 
-				responseTcs.SetException(e);
+				responseTcs!.SetException(e);
```

Comment:
The change in line 68 is enough to fix the warning here, so this change is not necessary AFAICT.

Reply:
Fine, for me I do not get those warnings.